lib/view/students/memberships_screen.dart

```diff
@@ -14,7 +15,7 @@ class MembershipsOfStudent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-      //fetching data
+    //fetching data
```

Comment:
Instead of having a comment saying "fetching data", how about having a function called "fetchMemberships"

Reply:
solved #14